charts/aws-vpc-cni/values.yaml

```diff
@@ -78,6 +78,7 @@ env:
   DISABLE_NETWORK_RESOURCE_PROVISIONING: "false"
   ENABLE_IPv4: "true"
   ENABLE_IPv6: "false"
+  VPC_CNI_VERSION: "v1.15.0"
```

Comment:
Based on what I saw from the implementation of other variables, it seems the other env variable values are eventually hardcoded within the code. This means when upgrading versions we'll have to manually update the default value when calling `GetEnv()` in `session.go`, which defeats the purpose of having this environment variable in the first place.

Reply:
If the environment variable is not defined, we can stick with the behavior as it is today, i.e. not including a version in the user agent string